design-system-docs/src/_component_examples/_radio_group/default.erb

```diff
@@ -0,0 +1,12 @@
+---
+title: Default
+---
+<%= render CitizensAdviceComponents::RadioGroup.new(
+  legend: "Example radio group",
+  name: "radio-buttons"
+) do |c|
+  c.inputs([
```

Comment:
Is there a way to reuse a radio group variable the same way it's done in `radio_group_preview.rb` (i.e `c.inputs(radios)`)? I found myself copy pasting the same inputs quite a lot.

Reply:
Possibly: You could define the options as front matter in `_component_examples/_radio_group/_defaults.yml` and use the data from each example. But I'm also fine with it being explicit for example. Makes copy-pasting the example from the docs less error prone.